src/palette/internal/palettecreator.cpp

```diff
@@ -352,10 +370,14 @@ PalettePtr PaletteCreator::newBarLinePalette(bool defaultPalette)
             int from, to;
             const char* userName;
         } spans[] = {
-            { BARLINE_SPAN_TICK1_FROM,  BARLINE_SPAN_TICK1_TO,  SymNames::userNameForSymId(SymId::barlineDashed) },
-            { BARLINE_SPAN_TICK2_FROM,  BARLINE_SPAN_TICK2_TO,  QT_TRANSLATE_NOOP("engraving/sym", "Tick barline 2") },  // Not in SMuFL
-            { BARLINE_SPAN_SHORT1_FROM, BARLINE_SPAN_SHORT1_TO, SymNames::userNameForSymId(SymId::barlineShort) },
-            { BARLINE_SPAN_SHORT2_FROM, BARLINE_SPAN_SHORT2_TO, QT_TRANSLATE_NOOP("engraving/sym", "Short barline 2") }, // Not in SMuFL
+            { BARLINE_SPAN_TICK1_FROM, BARLINE_SPAN_TICK1_TO,
+              SymNames::userNameForSymId(SymId::barlineTick) },
```

Comment:
This is the one and onle change that is really needed

Reply:
Hey Jojo!

Thanks - I agree I think my vim has formatted the file. I expected the uncrustify would format the file correctly but it hasn't! I will try and revert and turn off the formatting.